packages/swc-helpers/src/_object_spread_props.js

```diff
@@ -0,0 +1,30 @@
+function ownKeys(object, enumerableOnly) {
+  var keys = Object.keys(object);
+  if (Object.getOwnPropertySymbols) {
+    var symbols = Object.getOwnPropertySymbols(object);
+    if (enumerableOnly) {
+      symbols = symbols.filter(function (sym) {
+        return Object.getOwnPropertyDescriptor(object, sym).enumerable;
+      });
+    }
+    keys.push.apply(keys, symbols);
+  }
+  return keys;
+}
+
+export default function _objectSpreadProps(target, source) {
+  source = source != null ? source : {}
+  if (Object.getOwnPropertyDescriptors) {
+    Object.defineProperties(target, Object.getOwnPropertyDescriptors(source));
+  } else {
+    ownKeys(Object(source)).forEach(function (key) {
```

Comment:
I'm not sure if a browser without `Object.getOwnPropertyDescriptors` will have `Array.forEach`

Reply:
Oh, that would be a problem

Reply:
Oh, IE support `Array#forEach` starting with 9